README.md

```diff
@@ -20,6 +20,7 @@ Update the version in `vendir.s1.yml` and run `./sync/sync.sh`. It's good to che
 
 | Cilium app | Cilium version | CAPA | CAPZ | CAPV | CAPVCD |
 | --- | --- | --- | --- | --- | --- |
+| 1.3.x | 1.18.x | v31 | v31 | v31 | v31 |
```

Comment:
maybe we can land this on v32.0.0? Not sure what their timeline is

```suggestion
| 1.3.x | 1.18.x | v32 | v32 | v32 | v32 |
```

Reply:
thanks! I wanted to ask you about the meaning of the columns :D